LICENSE

```diff
@@ -1,4 +1,2 @@
 The Spinal HDL core is under the LGPL license
 The Spinal HDL lib is under the MIT license
-
-More details can be found there : http://spinalhdl.github.io/SpinalDoc/#license
```

Comment:
Thanks for catching this! Please update to the correct link: https://spinalhdl.github.io/SpinalDoc-RTD/master/SpinalHDL/Introduction/License.html#license instead of deleting it